src/mesh/python/x3d_generator.py

```diff
@@ -161,7 +161,8 @@
 # -- unused data per cell
 fo.write('cell_data\n')
 fo.write('matid\n')
-fo.write('         0\n')
+for cell in range(mesh.num_cells):
+    fo.write(str(cell+1)+'         0\n')
```

Comment:
It appears the Lint `flake8` check wants whitespace around the plus operators (sorry)!